code-challenges.test.js

```diff
@@ -23,37 +23,195 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+// Im thinking that I will have to loop through the array, maybe I can use the .map method, If not I will use a for loop
+// Once I get access to each item(element/value) in the array I will have to get access to the first letter in the string and then use the .toUpperCase() method
+// Im thinking that I will have to split the strings and access the frist index [0].toUpperCase()
+//Once I find some success with that, then I will return everything back into a nice loking array to match the expect output.
+//GL lol seems easy no prob guan
+
+const stringCap = (arr) => {
+    //I dont like how I can not see the console.log() here when testing with jest...
+
+    return arr.map((value) => {
+        //with a little digging,  I was able to find this built in method .substing()
+        //pretty nice and useful
+        return value[0].toUpperCase() + value.substring(1)
+    })
+}
+
+//Great News first test pass and no errors. Great Job Self
+
+// stingCap
+// ✓ Takes in an array of words and returns an array with all the words capitalized
+
+// Test Suites: 1 passed, 1 total
+
+
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
-// Expected output: 1
-const vowelTester2 = "academy"
-// Expected output: 0
-const vowelTester3 = "challenges"
-// Expected output: 2
+
+
+describe('firstVowelIndex', () => {
+    it("Takes in a string and logs the index of the first vowel", () => {
+        const vowelTester1 = "learn"
+        // Expected output: 1
+        const vowelTester2 = "academy"
+        // Expected output: 0
+        const vowelTester3 = "challenges"
+        // Expected output: 2
+
+        expect(firstVowelIndex(vowelTester1)).toEqual(1)
+        expect(firstVowelIndex(vowelTester2)).toEqual(0)
+        expect(firstVowelIndex(vowelTester3)).toEqual(2)
+    })
+})
+
+// firstVowelIndex
+// ✕ Takes in a string and logs the index of the first vowel
+
+// ● firstVowelIndex › Takes in a string and logs the index of the first vowel
+
+// ReferenceError: firstVowelIndex is not defined
 
 // b) Create the function that makes the test pass.
+
+// So I think the name of my function firstVowelIndex will give away the functions purpose. 
+// The function will take in a arguement/pram(string) and then see if it has any vowels and if so it will return the first instance of the that vowel and return the Index
+// I will have to split the string into an array using .split() (EDIT: I keep forgetting to add "" in the .split(''))
+// Then use a for loop to iterate through the string array and use a conditional to check for all vowels
+// The conditionals will use the && 'AND' and loosely or strictly compare each index for vowels
+// Once the first vowel is found, the conditional is happy and will return the index
+
+
+const firstVowelIndex = (string) => {
+    let stringArr = string.split('')
```

Comment:
Great work on this problem.  Just a side note, for loops can iterate on both strings and arrays, so there is actually no need in this instance to split the string first.  However, splitting the string would be necessary when using higher order functions.